### ISSUE TITLE
Merging with EJS Language Support

### ISSUE THREAD

**Issue by Betanu701** [theranbrig/ejs-snippets#2]
It looks like you have been doing some good work on this extension. If you are willing to, you can create a pull request and merge all your snippets into EJS Language support. Currently, it has over 52K downloads. If you rather keep it separate, that is fine as well. 

https://github.com/Digitalbrainstem/ejs-grammar

**Comment by the1abel** [comment]
@theranbrig , I like the suggestion made by @Betanu701 sounds great.  The complementary features of these two extensions is a match made in heaven!

**Comment by Betanu701** [comment]
@theranbrig I added this to my ejs-grammar. I mentioned you as the original author of the snippets. I did modify it though to include some better emmet functions. So one can use the original as you wrote, using ejs-> or can access the same features while they type and if they type the standard <% ejs tag like one would do in while writing html code, they get the correct popups. 

Thanks for your hard work!
